apps/catalogue/package.json

```diff
@@ -30,24 +30,25 @@
     "if-env": "^1.0.4",
     "jsdom": "22.1.0",
     "metadata-utils": "*",
+    "nuxt": "3.14.1592",
     "playwright-core": "1.50.0",
     "postcss": "8.4.39",
     "postcss-custom-properties": "13.3.11",
     "prettier": "2.8.8",
     "shelljs": "0.8.5",
     "std-env": "3.7.0",
+    "tailwind-components": "*",
     "tailwindcss": "3.4.4",
     "typescript": "5.6.2",
     "vitest": "2.1.9",
-    "vue-tsc": "2.1.6",
-    "nuxt": "3.14.1592",
-    "tailwind-components": "*"
+    "vue-tsc": "2.1.6"
   },
   "dependencies": {
     "@vueuse/core": "10.7.0",
     "floating-vue": "5.2.2",
     "metadata-utils": "*",
     "nuxt-gtag": "3.0.1",
+    "pinia": "^3.0.1",
```

Comment:
we do fixed versions, so the dependency bot will ask you to remove the ^ so you fix the version

Reply:
That's good to know. I've removed it.